samtranslator/internal/schema_source/aws_serverless_api.py

```diff
@@ -134,6 +134,7 @@ class Route53(BaseModel):
     IpV6: Optional[bool] = route53("IpV6")
     SetIdentifier: Optional[PassThroughProp]  # TODO: add docs
     Region: Optional[PassThroughProp]  # TODO: add docs
+    SeparateRecordSetGroup: Optional[bool]  # TODO: add docs
```

Comment:
If it's plural "separate record set groups", it just means record set groups are separated (whatever it means). If it's singular "separate record set group", my first question is "separate what record set group?".

So IMO either we should keep it singular but specify what record set group it separates, or keep it plural so it's more generically just saying record set groups are separated.

Reply:
Oh I see, so it's a single separate record set group for this API, but if it's diabled then it'll do some "optimizations" and join records from multiple APIs?

Reply:
exactly!